util/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/AttributesUtil.java

```diff
@@ -114,13 +117,13 @@ public static String attributesToQidoPath(Attributes attrs, String... includeFie
     if (nonEmptyKeys.contains(Tag.QueryRetrieveLevel)) {
       switch (attrs.getString(Tag.QueryRetrieveLevel)) {
         case "STUDY":
-          qidoPath.append("studies");
+          qidoPath.append("studies?limit=" + STUDIES_SERIES_LIMIT + "&");
```

Comment:
I think we should have an integration test step for studies and series C FIND requests since they're a different code path, can you please add this in a follow up PR?

Reply:
Ok, added.